vircadia/vircadia
# Text Entity alignment property

fix #221 resolve #206 

Adds an alignment property to text entities that can be “left” (default), “right”, or “center”.

Test plan:
- Create a text entity with text that spans multiple lines with spaces and various characters
- In Create, confirm that all three alignment options work as expected
- Verify that they still work with effects, different margins, when you resize the entity, etc
